tests/publiccloud/prepare_tools.pm

```diff
@@ -68,7 +73,7 @@ sub run {
     }
 
     # Install prerequesite packages test
-    zypper_call('-q in python3-pip python3-devel python3-virtualenv python3-img-proof python3-img-proof-tests');
+    zypper_call('-q in python3-pip python3-devel python3-virtualenv python3-img-proof python3-img-proof-tests podman');
```

Comment:
why we do not put it to autoyast profile?

Reply:
only reason that it is not goal of this PR , but yeah we need to do it someday ...

Reply:
I also like the idea of moving it to autoyast as we do for other packages, but ok to do it in another PR.